docs/en/setup/backend/backend-storage.md

```diff
@@ -8,325 +8,18 @@ storage:
-```
-All connection-related settings, including URL link, username, and password, are found in `application.yml`.
-Only part of the settings is listed here. Please follow [HikariCP](https://github.com/brettwooldridge/HikariCP) connection pool document for full settings.
-
-## BanyanDB
-[BanyanDB](https://github.com/apache/skywalking-banyandb) is a dedicated storage implementation developed by the SkyWalking Team and the community.
-Activate BanyanDB as the storage, and set storage provider to **banyandb**.
-
-```yaml
-storage:
-  banyandb:
-    host: ${SW_STORAGE_BANYANDB_HOST:127.0.0.1}
-    port: ${SW_STORAGE_BANYANDB_PORT:17912}
-    maxBulkSize: ${SW_STORAGE_BANYANDB_MAX_BULK_SIZE:5000}
-    flushInterval: ${SW_STORAGE_BANYANDB_FLUSH_INTERVAL:15}
-    metricsShardsNumber: ${SW_STORAGE_BANYANDB_METRICS_SHARDS_NUMBER:1}
-    recordShardsNumber: ${SW_STORAGE_BANYANDB_RECORD_SHARDS_NUMBER:1}
-    superDatasetShardsFactor: ${SW_STORAGE_BANYANDB_SUPERDATASET_SHARDS_FACTOR:2}
-    concurrentWriteThreads: ${SW_STORAGE_BANYANDB_CONCURRENT_WRITE_THREADS:15}
-    profileTaskQueryMaxSize: ${SW_STORAGE_BANYANDB_PROFILE_TASK_QUERY_MAX_SIZE:200} # the max number of fetch task in a request
-    streamBlockInterval: ${SW_STORAGE_BANYANDB_STREAM_BLOCK_INTERVAL:4} # Unit is hour
-    streamSegmentInterval: ${SW_STORAGE_BANYANDB_STREAM_SEGMENT_INTERVAL:24} # Unit is hour
-    measureBlockInterval: ${SW_STORAGE_BANYANDB_MEASURE_BLOCK_INTERVAL:4} # Unit is hour
-    measureSegmentInterval: ${SW_STORAGE_BANYANDB_MEASURE_SEGMENT_INTERVAL:24} # Unit is hour
-```
-
-For more details, please refer to the documents of [BanyanDB](https://skywalking.apache.org/docs/skywalking-banyandb/next/readme/)
-and [BanyanDB Java Client](https://github.com/apache/skywalking-banyandb-java-client) subprojects.
-
+- [H2](storages/h2.md)
+- [OpenSearch](storages/elasticsearch.md#opensearch)
+- [ElasticSearch 7 and 8](storages/elasticsearch.md#elasticsearch). 
+- [MySQL and its compatible databases](storages/mysql.md)
+- [PostgreSQL and its compatible databases](storages/postgresql.md)
+- [BanyanDB(alpha stage)](storages/banyandb.md)
+
+H2 is the default storage option in the distribution package. It is recommended to use H2 for testing and development ONLY.
+Elasticsearch and OpenSearch are recommended for production environments, specially for large scale deployments.
+MySQL and PostgreSQL are recommended for production environments for medium scale deployments, especially for low trace
+and log sampling rate. Some of their compatible databases may support larger scale better, such as TiDB and AWS Aurora.
+
+BanyanDB is going to be our next generation storage solution. It is still in alpha stage. It has shown high potential performance
+improvement. Less than 50% CPU usage and 50% memory usage compared to Elasticsearch in the same scale with 100% sampling.
+We are looking for early adoption, and it would be our first-class recommended storage option since 2024.
```

Comment:
@lujiajing1126 @hanahmily Welcome to provide more positive data for expected performance.
And I assume this date(since 2024) should be good for you both? The expectation is the cluster mode is ready(0.5) and the new file system is implemented(0.6). After that, we should announce the welcome information with a complete performance report.

What do you think about this? Please let me know.

Reply:
Updated to `Less than 50% CPU usage and 50% memory usage with 40% disk volume `.

We will have new tests in the cluster mode. Let's update those numbers after that.